nuxeo-runtime/nuxeo-stream/src/main/java/org/nuxeo/lib/stream/log/chronicle/ChronicleRetentionListener.java

```diff
@@ -107,10 +109,14 @@ protected void dropCycle(Integer cycle) {
     }
 
     protected File getFileForCycle(int cycle) {
```

Comment:
As this method is the one leaking the store and is not used anywhere else, could you refactor a bit more to completely remove it?